### ISSUE TITLE
t/11-timeout.t fails with non-English locale

### ISSUE THREAD

**Issue by eserte** [shogo82148/Redis-Fast#46]
I observe the following test failure if a German locale (LC_ALL=de_DE.UTF-8) is set:
```
    #   Failed test 'the code died as expected'
    #   at t/11-timeout.t line 32.
    #                   'Error while reading from Redis server: Die Ressource ist zur Zeit nicht verfÃ¼gbar at /tmpfs/.cpan-build/2015121006/Redis-Fast-0.16-tfk2Tt/blib/lib/Redis/Fast.pm line 212.
    # '
    #     doesn't match '(?^u:Error while reading from Redis server: Die Wartezeit für die Verbindung ist abgelaufen|Error while reading from Redis server: Die Ressource ist zur Zeit nicht verfügbar)'
    # Looks like you failed 1 test of 2.

#   Failed test 'server doesn't replies quickly enough'
#   at t/11-timeout.t line 35.
# Looks like you failed 1 test of 3.
t/11-timeout.t ............ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```

**Comment by shogo82148** [comment]
related https://github.com/PerlRedis/perl-redis/issues/111

**Comment by shogo82148** [comment]
#49 fixes this issue and Redis::Fast 0.17 has been released.
